sentry-samples/sentry-samples-logback/src/main/resources/logback.xml

```diff
@@ -11,13 +11,11 @@
       <!-- NOTE: Replace the test DSN below with YOUR OWN DSN to see the events from this app in your Sentry project/dashboard -->
       <dsn>https://f7f320d5c3a54709be7b28e0f2ca7081@sentry.io/1808954</dsn>
     </options>
-
-    <!-- Optionally you can filter what statements are sent to Sentry independently from loggers configuration -->
-<!--    <filter class="ch.qos.logback.classic.filter.ThresholdFilter">-->
-<!--      <level>WARN</level>-->
-<!--    </filter>-->
+    <minimumEventLevel>WARN</minimumEventLevel>
+    <minimumBreadcrumbLevel>DEBUG</minimumBreadcrumbLevel>
```

Comment:
```suggestion
    <!-- Demonstrates how to modify the minimum values -->
    <!-- Default for Events is ERROR -->
    <minimumEventLevel>WARN</minimumEventLevel>
    <!-- Default for Events is INFO -->
    <minimumBreadcrumbLevel>DEBUG</minimumBreadcrumbLevel>
```

Reply:
Current minimum is debug - anything that gets through to appender. Should I change it to `ERROR` for events and `INFO` for breadcrumbs?